src/utils/utils.ts

```diff
@@ -20,7 +36,7 @@ export const isInputElement = (activeElement: Element | null) => {
 
 export const getUrlParam = (paramName: string) => {
   const urlParams = new URLSearchParams(window.location.search);
-  return urlParams.get(paramName);
+  return urlParams.has(paramName);
 };
```

Comment:
with this change, it seems like the method should be renamed to `hasUrlParam` since it's no longer a getter.

Reply:
I just changed this back after consulting with Ethan and making a change to the url parameter we're checking for.